### PR TITLE
[MeshMap] Increase internal links

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**
- Used the new Rotational CTA for `Resources` and `Events`
 - Added CTA on `News` sidebar

Screenshot 2022-07-03 at 4 11 27 AM
Screenshot 2022-07-03 at 4 10 15 AM

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit aacca231ccb36ba869be202f67a7644f66ce9eaa at: https://62c0cc9b9c3edd18d87a715a--layer5.netlify.app

File: src/sections/Company/News-single/Sidebar.js [hunk 1]
```diff
@@ -58,13 +103,31 @@ const NewsSidebar = (props) => {
...
+            </h3>
+            <div className="chapter-link">
+              <Button primary title="Try Now!"
+                className="btn"
+                url="/meshmap"
+                external={false} />
+            </div>
+          </div>
+        </div>
+      </Link>
```
[the remark this answers is not in the document]
Comment (Nikhil-Ladha) [reply src/sections/Company/News-single/Sidebar.js:130]: Or maybe, `CTA_ImageOnly` ?

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Considering the image of MeshMap, I think it is better to opt for `opacity: 0.5` in `CTA_FullWidth` and `CTA_Bottom` for responsive views, instead of `filter: brightness(0.5)`. Can you please update that (both the places)?

Also, the content seems to be crossing the limits and moving beyond the boundaries! :p
What I mean to say is, after 992px the text is peeking outside the CTA box, that needs to be fixed as well.

Rest looks good.

File: src/components/Call-To-Actions/CTA_Bottom/index.js [hunk 2]
```diff
@@ -29,12 +30,13 @@ const CTA_BottomWrapper = styled.div`
         padding: 0.5rem;
         display: flex;
         flex: auto;
-        background: rgba(201, 252, 246, 0.3);
-        text-align: center;
+        text-align: left;
```
Comment (Nikhil-Ladha) [inline src/components/Call-To-Actions/CTA_Bottom/index.js:33]: Do we really need this?

File: src/components/Call-To-Actions/CTA_Bottom/index.js [hunk 3]
```diff
@@ -29,12 +30,13 @@ const CTA_BottomWrapper = styled.div`
...
         align-items: center;
         height: 100%;
 
         p {
+            margin-top: 0.5rem;
+            width: 95%;
             flex: 0 0 75%;
         }
         a {
```
Comment (Nikhil-Ladha) [inline src/components/Call-To-Actions/CTA_Bottom/index.js:42]: `a` needs a `text-align: center` as well.

**Comment by Nikhil-Ladha** [comment]
Having this CTA in action brought about a lot of small CSS issues, which got ignored earlier.
Sorry for the extra trouble @debo19 , hopefully you won't mid taking care of it :)

**Comment by leecalcote** [comment]
You get a CTA! You get a CTA! We all get CTAs!

**Comment by leecalcote** [comment]
The CTA in /resources is looking pretty good. I could be missing something (like the need for an increased amount of padding within the border of the CTA), but think this is good to go.

**Comment by l5io** [comment]
🚀 Preview for commit e18569d34733e1fb1ecdeed2c267022d2561f928 at: https://62cbcb7ded70286127d428ce--layer5.netlify.app

File: src/components/Call-To-Actions/CTA_Bottom/index.js [hunk 4]
```diff
@@ -11,50 +11,49 @@ const CTA_BottomWrapper = styled.div`
...
             min-width: 6.5rem;
         }
-    }
-
-    @media screen and (max-width: 699px) {
-        button {
-            min-width: auto;
+        img{
+          width: 16rem;
+          height: 16rem;
```
Comment (Nikhil-Ladha) [inline src/components/Call-To-Actions/CTA_Bottom/index.js:52]: add `align-self: center` here.

Comment (Nikhil-Ladha) [reply src/components/Call-To-Actions/CTA_Bottom/index.js:52]: Same applies for `CTA_Fullwidth`

**Comment by debo19** [comment]
@Nikhil-Ladha With different images, the styles for `CTA_Bottom` and `CTA_FullWidth` are not completely in accordance. While I've used the icon with a transparent background in the `CTA_Bottom` here, it required padding and the other changes you suggested. However, if we use one like the default `community` image, your previous styles work well. Should we look into the styles for `CTA_FullWidth` when we actually put it to use? 

Screenshot 2022-07-14 at 1 36 10 AM

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha With different images, the styles for `CTA_Bottom` and `CTA_FullWidth` are not completely in accordance. While I've used the icon with a transparent background in the `CTA_Bottom` here, it required padding and the other changes you suggested. However, if we use one like the default `community` image, your previous styles work well. Should we look into the styles for `CTA_FullWidth` when we actually put it to use?
> 
> Screenshot 2022-07-14 at 1 36 10 AM

Sure, we can handle the other one later on.

**Comment by l5io** [comment]
🚀 Preview for commit 299548575ba250c70d2f98a7d240f7a45ac7d2a7 at: https://62d1b2bf0eebee5ff09d4624--layer5.netlify.app

**Comment by debo19** [comment]
Added one CTA to blogs too.

**Review by Nikhil-Ladha — APPROVED** [review]
Looking good!

**Pull request merged**